Water Soil Wetness Prediction

deepanshudutta743 opened the pull request:
With the help of this CNN model you can predict soil wetness either soil is dry,wet or extremely wet.
#156

deepanshudutta743 committed:
Water Soil Wetness Prediction

With the help of this CNN model you can predict soil wetness either soil is dry,wet or extremely wet.

ashishsahu1 commented:
Good one @dashingdiptu12345 Few things you can add
- add pkl file for your model

deepanshudutta743 commented:
> Good one @dashingdiptu12345 Few things you can add
> 
> * add pkl file for your model

ok sir.

deepanshudutta743 committed:
Delete Water Soil Wetness Prediction directory

deepanshudutta743 committed:
Soil Wetness prediction

With the help of this CNN model you can predict  either soil is dry,wet or extremely wet

deepanshudutta743 commented:
Sir I have added the pickle file please check the pr.

ashishsahu1 commented:
I think it's good to go
